customer/src/test/java/nl/softwarestrijders/waiter/customer/domain/AddressTest.java

```diff
@@ -0,0 +1,129 @@
+package nl.softwarestrijders.waiter.customer.domain;
+
+import nl.softwarestrijders.waiter.customer.domain.exceptions.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AddressTest {
+	private int houseNumber;
+	private String addition;
+	private String street;
+	private String postalCode;
+	private String city;
+
+	@BeforeEach
+	public void initialize() {
+		this.houseNumber = 1;
+		this.addition = "A";
+		this.street = "TestStreet";
+		this.postalCode = "1010AB";
+		this.city = "testCity";
+	}
+
+	@Test
+	@DisplayName("Should create Address successfully")
+	void shouldCreateAddressSuccessfully() {
+		assertDoesNotThrow(() -> new Address(houseNumber, addition, street, postalCode, city));
+	}
+
+	@Test
+	@DisplayName("Should create Address successfully when street contains whitespaces")
+	void shouldCreateAddressSuccessfullyWithSpaceInStreet() {
+		assertDoesNotThrow(() -> new Address(houseNumber, addition, "Test street", postalCode, city));
+	}
+
+	@Test
+	@DisplayName("Should throw InvalidHouseNumberException if housenumber is invalid")
+	void shouldThrowWhenInvalidHouseNumber() {
+		assertThrows(InvalidHouseNumberException.class, () -> new Address(- 1, addition, street, postalCode, city));
+	}
+
+	@Test
+	@DisplayName("Should throw if addition is invalid")
+	void shouldTrowWhenInvalidAddition() {
+		assertThrows(InvalidAdditionException.class, () -> new Address(houseNumber, "-", street, postalCode, city));
+	}
+
+	@Test
+	@DisplayName("Should throw if addition contains whitespaces")
```

Comment:
```suggestion
	@DisplayName("Should not throw if addition contains whitespaces")
```

Tbh i don't think these displaynames add anything, it's saying the same stuff